src/Interfaces/ISDK.ts

```diff
@@ -17,6 +17,6 @@ export default interface ISDK {
   emailTranscript(requestId: string, token: string, emailRequestBody: object): Promise<void>;
   fetchDataMaskingInfo(requestId: string): Promise<IDataMaskingInfo>;
   makeSecondaryChannelEventRequest(requestId: string, secondaryChannelEventRequestBody: object): Promise<void>;
-  getQueueAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams): Promise<object>
+  getAgentAvailability(requestId: string, queueAvailabilityOptionalParams: IGetQueueAvailabilityOptionalParams): Promise<object>
```

Comment:
Rename `queueAvailabilityOptionalParams` & `IGetQueueAvailabilityOptionalParams` accordingly

Reply:
Would be renaming internal method signatures with next phase as this was a last moment requirement so changing only the once exposed to customer.
[Task 2625527](https://dev.azure.com/dynamicscrm/OneCRM/_workitems/edit/2625527): Renaming GetQueueAvailability to AgentAvailability at internal signatures of API